types/floreal/floreal-tests.ts

```diff
@@ -1,10 +1,13 @@
-import { Date } from 'floreal';
+import { Date as Floreal } from "floreal";
 
-const fd1 = new Date();
-const fd2 = new Date('1799-11-09');
+const fd1 = new Floreal(); // $ExpectType FlorealDate
+const fd2 = new Floreal("1799-11-09"); // $ExpectType FlorealDate
+const fd3 = new Floreal(Date.now()); // $ExpectType FlorealDate
```

Comment:
`Date.now()` returns a number, not a `Date`, so I guess this should be
```
const fd3 = new Floreal(new Date());
```
in order to test the functionality we intend to be testing

Reply:
🤦

Reply:
amended, thx!